src/Dfe.Spi.GraphQlApi.Application/GraphTypes/LearningProviderType.cs

```diff
@@ -10,20 +10,20 @@ public LearningProviderType()
             Field(x => x.Name)
                 .Name("name")
                 .Description("Name of the learning provider");
-            
-            Field(x => x.LegalName)
+
+            Field(x => x.LegalName, nullable:true)
                 .Name("legalName")
                 .Description("Legal name of the learning provider");
             
-            Field(x => x.Urn)
+            Field(x => x.Urn, nullable:true)
                 .Name("urn")
                 .Description("URN of the learning provider");
             
-            Field(x => x.Ukprn)
+            Field(x => x.Ukprn, nullable:true)
                 .Name("ukprn")
                 .Description("UKPRN of the learning provider");
             
-            Field(x => x.Postcode)
+            Field(x => x.Postcode, nullable:true)
                 .Name("postcode")
                 .Description("Postcode of the learning provider");
         }
```

Comment:
Do you reckon it may be possible to populate this at run-time via reflection? Or maybe via some sort of configuration mechanism?

Just looking at this on the surface of it; it may not be; just let me know if that's the case.

Reply:
Perhaps. Probably worth thinking about on the next round of fields